mmcv/runner/hooks/logger/text.py

```diff
@@ -37,21 +42,39 @@ def log(self, runner):
             log_str += (
                 'time: {log[time]:.3f}, data_time: {log[data_time]:.3f}, '.
                 format(log=runner.log_buffer.output))
-        # statistic memory
-        if runner.mode == 'train' and torch.cuda.is_available():
-            mem = torch.cuda.max_memory_allocated()
-            mem_mb = torch.tensor([mem / (1024 * 1024)],
-                                  dtype=torch.int,
-                                  device=torch.device('cuda'))
-            if runner.world_size > 1:
-                dist.reduce(mem_mb, 0, op=dist.ReduceOp.MAX)
-            log_str += 'memory: {}, '.format(mem_mb.item())
+        json_log = dict()
+        json_log['mode'] = mode
+        json_log['epoch'] = runner.epoch + 1
+        if mode == 'train':
+            json_log['iter'] = runner.inner_iter + 1
+            json_log['lr'] = float(lr_str)
+            json_log['time'] = round(runner.log_buffer.output['time'], 4)
+            json_log['data_time'] = round(
+                runner.log_buffer.output['data_time'], 4)
```

Comment:
The processing of `json_log` and the printed info is somewhat duplicated. They can share the same dict and just different outputs.